### DISCUSSION TITLE
Class 'SoundStreamPlugin' is not abstract and does not implement abstract member public abstract fun onRequestPermissionsResult(

### DISCUSSION THREAD

**Discussion by medicalnp** [deepgram/community#183] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

 C:\Flutter\.pub-cache\hosted\pub.dartlang.org\sound_stream-0.3.0\android\src\main\kotlin\vn\casperpas\sound_stream\SoundStreamPlugin.kt: (45, 8): Class 'SoundStreamPlugin' is not abstract and does not implement abstract member public abstract fun onRequestPermissionsResult(p0: Int, p1: Array<(out) String!>, p2: IntArray): Boolean defined in io.flutter.plugin.common.PluginRegistry.RequestPermissionsResultListener
e: C:\Flutter\.pub-cache\hosted\pub.dartlang.org\sound_stream-0.3.0\android\src\main\kotlin\vn\casperpas\sound_stream\SoundStreamPlugin.kt: (182, 5): 'onRequestPermissionsResult' overrides nothing

I have followed the exact tutorial for flutter speech to text - as per the url below. 
https://blog.deepgram.com/flutter-speech-to-text-tutorial/

**Comment by jjmaldonis** [comment, marked as answer]
Hey @medicalnp apologies for the late reply on this. We updated our Flutter tutorial on Monday so this issue should be fixed now. Here's the new link to the blog: https://deepgram.com/learn/flutter-speech-to-text-tutorial
